zzz_generated/all-clusters-app/zap-generated/endpoint_config.h

```diff
@@ -2469,301 +2471,834 @@
         (EmberAfGenericClusterFunction) MatterIasZoneClusterServerPreAttributeChangedCallback,                                     \
     };
 
+#define GENERATED_COMMANDS                                                                                                         \
+    { /* Endpoint: 0, Cluster: Identify (server) */                                                                                \
+        0x00000000 /* Identify */, 0x00000001 /* IdentifyQuery */, 0x00000040 /* TriggerEffect */, 0xFFFF'FFFF /* end of list */,  \
```

Comment:
I wonder whether joining each list on `",\\\n"` would make this a bit more readable, or whether the reformatter would make this soup out of it anyway.  I would hope it would not...

Reply:
Will add // clangformat off to disable formatter here.

Reply:
Then this will all be one huge line, no?

Reply:
Updated